### DISCUSSION TITLE
How to display Private Contributions

### DISCUSSION THREAD

**Discussion by infm5** [Ashutosh00710/github-readme-activity-graph#137] [Ideas]
# Private Contributions

Is there any way to enable displaying **Private Contributions?**

**Comment by Kshitij978** [comment]
Hi @deepraj02 It does show the contributions on private repositories. Did you enable the `repo` option while creating your personal token?

![image](https://user-images.githubusercontent.com/42491256/194744725-465c0b09-b344-4271-9970-357d29217dd7.png)

**Comment by ysw0121** [comment]
Hi @deepraj02  I just turned on the 2 things in contribution settings, and found the graph showed my private commits.

![image](https://github.com/user-attachments/assets/7fb7f6af-9be3-4352-9fc5-9ce503cc3c38)

![image](https://github.com/user-attachments/assets/2bdedcfe-e1e2-4cf2-840f-a93a0987c96f)
